src/modals/WordsModal.svelte

```diff
@@ -6,15 +6,15 @@
 	 */
 	let privateKeyMnemonic: string[] = [];
 	const nimiqWallet = $wallet;
-	privateKeyMnemonic = Nimiq.MnemonicUtils.entropyToMnemonic(
+	privateKeyMnemonic = Nimiq.MnemonicUtils.entropyToLegacyMnemonic(
 		nimiqWallet.keyPair.privateKey.serialize(),
 	);
 	export const mnemonic = privateKeyMnemonic.join(" ");
 
 	const saveMnemonic = () => {
 		let text = "";
```

Comment:
Suggestion:
```typescript
const text = privateKeyMnemonic.map(word => `${word}\n`)
```

Reply:
O now that I think, we could also use `typescript privateKeyMnemonic.join('\n')`

Reply:
then we are sticking with the `.foreach`?

Reply:
I've changed it but on saveMnemonic instead of where you suggested it